web/ui/react-app/src/App.css

```diff
@@ -212,6 +212,19 @@ div.time-input {
     margin-bottom: 20px;
 }
 
+.label-button-style {
+    margin-bottom: 1%;
+}
+
+.target-head {
+    font-weight: 700;
+    font-size: large;
+}
+
+.label-component {
+    margin-bottom: 1%;
```

Comment:
I don't see any difference when removing this (only when I use larger percentages), is this style rule needed?